code/game/machinery/cryo.dm

```diff
@@ -10,11 +13,11 @@
 	interact_offline = 1
 	max_integrity = 350
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 100, "bomb" = 0, "bio" = 100, "rad" = 100, "fire" = 30, "acid" = 30)
-	var/on = 0
+	var/on = FALSE
 	var/temperature_archived
 	var/mob/living/carbon/occupant = null
 	var/obj/item/reagent_containers/glass/beaker = null
-	var/autoeject = 0
+	var/auto_eject_prefs = 0
```

Comment:
small thing. You can use NONE here to make it denote it holds something and is not a number value.